ethbloom/src/lib.rs

```diff
@@ -204,16 +200,16 @@ impl Bloom {
 pub struct BloomRef<'a>(&'a [u8; BLOOM_SIZE]);
 
 impl<'a> BloomRef<'a> {
-	pub fn is_empty(&self) -> bool {
+	pub fn is_empty(self) -> bool {
```

Comment:
Hm, it that technically a breaking change?

Reply:
> #[derive(Clone, Copy)]
pub struct BloomRef<'a>(&'a [u8; BLOOM_SIZE]);

I don't think so, because struct `BloomRef` is **Copyable** semantically.

Reply:
I'm afraid if the consuming code uses UFCS like https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f62156c06e7e27e0ba1330c4e71f5879, it's will break the code.

Reply:
Yes, you're right. It will break the code :(
I will recover it.